front-end/src/components/Chat/ChatWindow.module.css

```diff
@@ -13,3 +14,18 @@
 
   border-radius: 10px;
 }
+.roomTitle{
+  color: var(--text-color);
+}
+.participant{
+  padding-top: 0.3rem;
+  color: var(--text-color);
+  font-weight: 200;
+}
+.roomHeading{
+  background-color: #232526;
```

Comment:
The standard colors(including this) are already defined in `front-end/src/App.css`. Use that instead